src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.spec.ts

```diff
@@ -1149,8 +1151,10 @@ describe('PoolFormComponent', () => {
         setMultipleValues({
           mode: 'passive',
           algorithm: 'lz4',
-          minBlobSize: '4 K',
-          maxBlobSize: '4 M',
+          minBlobSize: '4',
+          minBlobSizeUnit: 'KiB',
+          maxBlobSize: '4',
+          maxBlobSizeUnit: 'MiB',
           ratio: 0.7
         });
```

Comment:
```suggestion
        });
          fixture.detectChanges();
```
This will fix teh failing tests.